src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -1,7 +1,8 @@
 <div
   class="sky-input-group"
   [ngClass]="{
-    'sky-colorpicker-hidden': !isVisible
+    'sky-colorpicker-hidden': !isVisible,
+    'disabled': disabled
```

Comment:
We'll need to namespace our classes like so: `sky-colorpicker-disabled`, and `sky-colorpicker-button-disabled` (for the button below).

Reply:
done in last commit